packages/daisyui/src/templates/ObjectFieldTemplate/ObjectFieldTemplate.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  canExpand,
+  descriptionId,
+  FormContextType,
+  getTemplate,
+  getUiOptions,
+  ObjectFieldTemplateProps,
+  RJSFSchema,
+  StrictRJSFSchema,
+  titleId,
+} from '@rjsf/utils';
+
+export default function ObjectFieldTemplate<
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any
+>(props: ObjectFieldTemplateProps<T, S, F>) {
+  const {
+    description,
+    title,
+    properties,
+    required,
+    disabled,
+    readonly,
+    uiSchema,
+    idSchema,
+    schema,
+    formData,
+    onAddClick,
+    registry,
+  } = props;
+  const uiOptions = getUiOptions<T, S, F>(uiSchema);
+  const TitleFieldTemplate = getTemplate<'TitleFieldTemplate', T, S, F>('TitleFieldTemplate', registry, uiOptions);
+  const DescriptionFieldTemplate = getTemplate<'DescriptionFieldTemplate', T, S, F>(
+    'DescriptionFieldTemplate',
+    registry,
+    uiOptions
+  );
+  // Button templates are not overridden in the uiSchema
+  const {
+    ButtonTemplates: { AddButton },
+  } = registry.templates;
+
+  return (
+    <div className='form-control'>
+      {title && (
+        <TitleFieldTemplate
+          id={titleId<T>(idSchema)}
+          title={title}
+          required={required}
+          schema={schema}
+          uiSchema={uiSchema}
+          registry={registry}
+        />
+      )}
+      {description && (
+        <DescriptionFieldTemplate
+          id={descriptionId<T>(idSchema)}
+          description={description}
+          schema={schema}
+          uiSchema={uiSchema}
+          registry={registry}
+        />
+      )}
+      <div className={`grid grid-cols-1 gap-${description ? 2 : 6} mb-4`}>
+        {properties.map((element, index) =>
+          element.hidden ? (
+            element.content
+          ) : (
+            <div key={`${idSchema.$id}-${element.name}-${index}`}>{element.content}</div>
+          )
+        )}
+        {canExpand<T, S, F>(schema, uiSchema, formData) && (
+          <div className='flex justify-end'>
+            <AddButton
+              className='btn btn-primary btn-sm'
```

Comment:
Can you add `id={buttonId<T>(id, 'add')}` to match changes made in other themes in v6